/**:
  ros__parameters:
    mass: 0.82
    trajectory_control:
      antiwindup_cte: 1.0
      alpha: 0.1
      kp:
        x: 6.0
        y: 6.0
        z: 6.0
      ki:
        x: 0.005
        y: 0.005
        z: 0.065
      kd:
        x: 1.5
        y: 1.5
        z: 3.0
      roll_control:
        kp: 5.5
      pitch_control:
        kp: 5.5
      yaw_control:
        kp: 2.0
